hack/swagger-doc/build.gradle

```diff
@@ -0,0 +1,18 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'io.github.robwin:swagger2markup:0.6.0'
```

Comment:
Yeah, I'm worried about the dependencies of this PR as well, that's why I point them out in the PR message.
@bgrant0607, are we going to build these docs in every build, or are they going to be built only for every release?